.github/workflows/ci-test-frontend.yml

```diff
@@ -17,18 +17,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
 
       - uses: actions/setup-node@v1
 
       - run: npm ci
         working-directory: ./frontend
 
-      - run: npx run-p check lint lint:style test
-        working-directory: ./frontend
-
-      - run: npm run size
```

Comment:
why size and check:translation are gone?

Reply:
[Because](https://github.com/umputun/remark42/pull/679/files#diff-0a6694dd0d6b93a2aa14f32e3d873b4cR10) `check` runs all `check:*` commands. The same thing with `lint`